open_spiel/algorithms/mcts.cc

```diff
@@ -217,7 +218,8 @@ MCTSBot::MCTSBot(const Game& game, std::shared_ptr<Evaluator> evaluator,
       dirichlet_epsilon_(dirichlet_epsilon),
       rng_(seed),
       child_selection_policy_(child_selection_policy),
-      evaluator_(evaluator) {
+      evaluator_(evaluator),
+      dont_return_chance_node_{dont_return_chance_node} {
```

Comment:
Use of {...} is inconsistent with surround lines. Consistency is preferable.

Reply:
Use (...) instead